recipes/gtk/all/conanfile.py

```diff
@@ -59,42 +67,53 @@ def configure(self):
             raise ConanInvalidConfiguration("GTK recipe is not yet compatible with Windows. Contributions are welcome.")
 
     def build_requirements(self):
-        self.build_requires("meson/0.54.2")
+        self.build_requires("meson/0.56.2")
         self.build_requires("pkgconf/1.7.3")
+        if self._gtk4:
+            self.build_requires("sassc/3.6.1")
 
     def requirements(self):
```

Comment:
This is supper complicated... could we do this...

Or maybe just have two different functions? these two trees don't mix well

🤯 

```py
    def requirements(self):
        self.requires("gdk-pixbuf/2.42.0")
        self.requires("glib/2.67.0")
        self.requires("libepoxy/1.5.4")
        if self.options.with_pango:
            self.requires("pango/1.48.0")

        if self.settings.os == "Linux" and self.options.with_x11:
                self.requires("xorg/system"):

        if self.settings.compiler != "Visual Studio":
            self.requires("cairo/1.17.2")

        if self._gtk3:
            self.requires("atk/2.36.0")
            if self.settings.os == "Linux":
                self.requires("at-spi2-atk/2.38.0")

        if self._gtk4:
            self.requires("graphene/1.10.2")
            if self.settings.os == "Linux":
                self.requires("xkbcommon/1.0.3")

        if self.settings.os == "Linux":
            if self.options.with_wayland:
                if self._gtk3:
                    self.requires("xkbcommon/1.0.3")
                self.requires("wayland/1.18.0")
```

Reply:
I feel this is a situation where all the solutions are equally bad :/